packages/pds-ember/app/styles/pds/core/typography/heading/_config.scss

```diff
@@ -0,0 +1,21 @@
+$_module: "PDS.Core.Typography.Heading";
+/* --- [debug] CONFIG: #{$_module} --- */
+@use "../config" as Typography;
+
+@mixin appearance {
+  /* [debug] #{$_module}@appearance */
+
+  // Heading Variants (0-4)
+  @for $n from 0 through 4 {
+    &.pds--#{$n} {
```

Comment:
so this will be `.pds-heading.pds--0`, etc.?

Reply:
yes

Reply:
These classes will be handled by a yet-to-be-built modifier.